workflow/rules/post_dl3.smk

```diff
@@ -0,0 +1,217 @@
+ruleorder: plot_flux_points > plot_dl4
+ruleorder: plot_theta > plot_dl4
+
+
+# Plots using dl3 files
+rule observation_plots:
+    input:
+        build_dir / "dl3/hdu-index.fits.gz",
+        config=config_dir / "{analysis}/analysis.yaml",
+        script="scripts/events.py",
+    output:
+        build_dir / "plots/{analysis}/observation_plots.pdf",
+    resources:
+        mem_mb=64000,
+    conda:
+        gammapy_env
+    shell:
+        """
+        python {input.script} \
+            -c {input.config} \
+            -o {output} \
+        """
+
+
+rule calc_theta2_per_obs:
+    output:
+        build_dir / "dl3/theta2_{run_id}.fits.gz",
+    input:
+        data=build_dir / "dl3/dl3_LST-1.Run{run_id}.fits.gz",
+        script="scripts/calc_theta2_per_obs.py",
+        config=data_selection_config_path,
+        index=build_dir / "dl3/hdu-index.fits.gz",
+    wildcard_constraints:
+        run_id="\d+",
```

Comment:
Is this still needed? If not, I prefer to remove it, as otherwise it might get confusing in the future.  If yes, please add a comment explaining.

Reply:
Its needed to avoid the "theta2_stacked" to match this rule